apps/user-ms/src/main/java/it/pagopa/selfcare/user/util/UserUtils.java

```diff
@@ -41,6 +45,17 @@ public class UserUtils {
     private final ProductService productService;
     private final NotificationMapper notificationMapper;
     public static final List<String> VALID_USER_PRODUCT_STATES_FOR_NOTIFICATION = List.of(OnboardedProductState.ACTIVE.name(), OnboardedProductState.DELETED.name(), OnboardedProductState.SUSPENDED.name());
+    private static final String ACTIONS_FILE_PATH = "src/main/resources/role_action_mapping.json";
+
+    public Map<PartyRole, List<String>> retrieveActionsMap() {
+        try {
+            byte[] jsonFile = Files.readAllBytes(Paths.get(ACTIONS_FILE_PATH));
+            ObjectMapper objectMapper = new ObjectMapper();
+            return objectMapper.readValue(jsonFile, new TypeReference<>() {});
+        } catch (Exception e) {
+            throw new InvalidRequestException("Error during retrieve actions map");
+        }
+    }
```

Comment:
you could avoid to load every time the file but it might be better to load at startup and inject the map as bean